src/olympia/addons/models.py

```diff
@@ -1818,11 +1818,19 @@ def to_static_category(self):
         return staticcategory
 
     @classmethod
-    def from_static_category(cls, static_category):
+    def from_static_category(cls, static_category, save=False):
         """Return a Category instance created from a StaticCategory.
 
-        Does not save it into the database. Useful in tests."""
-        return cls(**static_category.__dict__)
+        Does not save it into the database by default. Useful in tests."""
+        # we need to drop description as it's a StaticCategory only property.
+        _dict = dict(static_category.__dict__)
+        del _dict['description']
```

Comment:
you probably will need to deepcopy the dict to avoid `static_category.__dict__` to be modified. Also add a test that ensures that.